tests/test_wallet.py

```diff
@@ -214,46 +214,44 @@ def test_get_transactions(self):
         transactions = private_key.get_transactions()
         assert transactions == private_key.transactions
 
-    # LEGACYADDRESSDEPRECATION
-    def test_send_cashaddress(self):
-        private_key = PrivateKeyTestnet(WALLET_FORMAT_COMPRESSED_TEST)
-        private_key.get_unspents()
-
-        initial = len(private_key.get_transactions())
-        current = initial
-        tries = 0
-        # FIXME: Changed jpy to satoshi and 1 to 10,000 since we don't yet
-        # have a rates API for BCH in place.
-        private_key.send([(BITCOIN_CASHADDRESS_TEST, 10000, 'satoshi')])
-
-        while tries < 15:  # pragma: no cover
-            current = len(private_key.get_transactions())
-            if current > initial:
-                break
-            time.sleep(5)
-            tries += 1
-
-    def test_send(self):
-        private_key = PrivateKeyTestnet(WALLET_FORMAT_COMPRESSED_TEST)
-        private_key.get_unspents()
-
-        initial = private_key.balance
-        current = initial
-        tries = 0
-        # FIXME: Changed jpy to satoshi and 1 to 10,000 since we don't yet
-        # have a rates API for BCH in place.
-        private_key.send([('n2eMqTT929pb1RDNuqEnxdaLau1rxy3efi', 10000, 'satoshi')])
-
-        while tries < 10:  # pragma: no cover
-            private_key.get_unspents()
-            current = private_key.balance
-            if current > initial:
-                break
-            time.sleep(20)
-            tries += 1
-
-        logging.debug('Current: {}, Initial: {}'.format(current, initial))
-        assert current < initial
+    # FIXME
+    # def test_send_cashaddress(self):
+    #     private_key = PrivateKeyTestnet(WALLET_FORMAT_COMPRESSED_TEST)
+    #     private_key.get_unspents()
+
+    #     initial = len(private_key.get_transactions())
+    #     current = initial
+    #     tries = 0
+    #     private_key.send([(BITCOIN_CASHADDRESS_TEST, 10000, 'satoshi')])
+
+    #     while tries < 15:  # pragma: no cover
+    #         current = len(private_key.get_transactions())
+    #         if current > initial:
+    #             break
+    #         time.sleep(5)
+    #         tries += 1
+
+    # def test_send(self):
```

Comment:
Is it not possible to have test_sent() working? It is a pretty good test to have.

Not sure what issues you ran into here with it.

Reply:
Should I merge this as is or do would you like to keep looking into test_send?

Looking good all in all! Just tend to think that if the error isn't obvious when a test fails, we don't have enough logging. If the test fails, it could be broken in some way that needs fixing anyways. But may just be worth merging anyway, up to you.

Reply:
I will keep looking today, if I can't find what the problem is, let's merge this and fix this in a future PR.

Reply:
Sounds good! Works for me. Don't be afraid to add logging statements, log.debug, log.info, etc, as needed. Add logs show up in the tests.

Reply:
Fixed! 😄